docs/deploy_narrative.md

```diff
@@ -51,14 +53,14 @@ Once a pull request is merged from `develop` to `master` a new _production_ imag
     -t IMAGE_TAG is the *current* Docker image tag, typically `pr-#` or `latest`
     -e TARGET Sets target environment's tag. This is typically either:`appdev`, `next`, or `prod`.
 ```
-4. Restart the `narrative` container for the desired environment (e.g. `appdev`) in Rancher.
+4. Upgrade the `narrative` container for the desired environment (e.g. `next` ) in Rancher.
 
 
 ### Image URLs
 
 | Evironment  | Image URL   |
```

Comment:
Evironment -> Environment (I know it's not part of your changes, but I just stumbled on it...